fwoa-tools/src/binaryConverter.ts

```diff
@@ -63,10 +67,6 @@ async function getBinaryResource(itemKey: string): Promise<GetObjectOutput> {
 
 // Step 3, Retrieve Binary objects from S3 Binary Bucket
 async function getBinaryObject(itemKey: string, versionId: number = 1): Promise<GetObjectOutput> {
```

Comment:
Ah I see where step 2 and 3 are. I would label step 2 and 3 in the main function `retrieveBinaryIdsFromFolder`, since that's where the ordered steps are happening. `getBinaryObject` can exist on it's own and so it's not technically a "step".